core/camel-core/src/test/java/org/apache/camel/component/file/strategy/FileChangedReadLockMinAgeTest.java

```diff
@@ -79,4 +79,4 @@ public void configure() throws Exception {
             }
         };
     }
-}
```

Comment:
I think you can remove this empty changes for this file and the other ones similar to this.